app/forms/declaration/CommodityMeasure.scala

```diff
@@ -29,14 +29,14 @@ object CommodityMeasure {
 
   val mapping = Forms.mapping(
     "supplementaryUnits" -> optional(
-      text().verifying("supplementary.commodityMeasure.supplementaryUnits.error", validateDecimal(16)(6) and containsNotOnlyZeros)
+      text().verifying("supplementary.commodityMeasure.supplementaryUnits.error", validateDecimalGreaterThanZero(17)(7) and containsNotOnlyZeros)
     ),
     "grossMass" -> text
       .verifying("supplementary.commodityMeasure.grossMass.empty", nonEmpty)
-      .verifying("supplementary.commodityMeasure.grossMass.error", isEmpty or validateDecimal(16)(6) and containsNotOnlyZeros),
+      .verifying("supplementary.commodityMeasure.grossMass.error", isEmpty or validateDecimalGreaterThanZero(17)(7) and containsNotOnlyZeros),
```

Comment:
In all cases it should be `(16)(6)`. Because We should use `<=` in the validation method.

Reply:
Patryk's suggestion is a very good one. Let's use the same data/numbers as the XML Schema, to avoid confusion in the future, between business analysts, developers and QAs

Reply:
Using the `17` was also my suggestion :stuck_out_tongue:

Reply:
done